ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/ModelServer.resource

```diff
@@ -201,10 +201,22 @@ Query Model With GRPCURL
           # Log    ${query_process.stdout}    console=yes
           Log    ${response}    console=yes
           # ${json_res}=    Load Json String    ${query_process.stdout}
-          ${json_res}=    Load Json String    ${response}
-          # ...    strict=False
-          RETURN    ${json_res}
+          IF    ${skip_res_json} == ${TRUE}
+            Log    ${response}
+            RETURN    ${response}
+          ELSE
+            ${json_res}=    Load Json String    ${response}
+            # ...    strict=False
+            RETURN    ${json_res}
+          END
     ELSE
           ${rc}    ${response}=    Run And Return Rc And Output    ${cmd}&
           Run Keyword And Continue On Failure    Should Be Equal As Integers    ${rc}    ${0}
     END
+
+Enable Toleration Feature In KNativeServing
+    [Documentation]    Enables the usage of node tolerations in inference services
+    [Arguments]    ${knative_serving_ns}
+    ${rc}    ${out}=    Run And Return Rc And Output
```

Comment:
I see this pattern used often. How about having a keyword that runs the command and checks return code and output?

```
Run And Compare RC And Output
    [Documentation]    Runs a command. Fails if return code doesn't match ${expected-rc} or output doesn't match  ${expected-output} 
    [Arguments]   ${command}    ${expected-rc}:0    ${expected-output}:${EMPTY}
```

Reply:
true. We can have that, similar to what @manosnoam did with Run And Watch.

It's a good improvement to have in next PRs, thanks for bringing this up